downstream/modules/platform/con-aap-installation-prereqs.adoc

```diff
@@ -4,7 +4,7 @@
 
 [role="_abstract"]
 
-* You chose and obtained a platform installer from the link:https://access.redhat.com/downloads/content/480/ver=2.1/rhel---8/2.1/x86_64/product-software[Red Hat Ansible Automation Platform Product Software].
+* You chose and obtained a platform installer from the link:https://access.redhat.com/downloads/content/480/ver=2.3/rhel---9/2.3/x86_64/product-software[Red Hat Ansible Automation Platform Product Software].
```

Comment:
Hi Donna, 
I'm just curious as to why you didn't use the attribute here? 
Otherwise it all looks good to me! 
Thanks,
Lizzi

Reply:
Hi @emurtoug , Good catch, I missed that one and will fix it. Thanks for the review!